### ISSUE TITLE
New features: Edit clipboard & view recent copied items

### ISSUE THREAD

**Issue by kashifo** [johanhil/copy-to-clipboard#5]
Hi there!
I would like to add these features ->
❶ Showing app in launcher to->
❷ View list of copied items
❸ Edit clipboard & update

 they will change the app lot. So shall i do them?

**Comment by johanhil** [comment]
Aren't there other applications that do this? Maybe its easier to add this
Share functionality to them, instead of re-writing this app to these
features.

It's up to you though :-)

**Comment by kashifo** [comment]
Yes, there are closed-source apps that do these, but not any of many open-source apps. that's why i want to do
